index.html

```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
                         <a href=""><img src="./images/Linkedin.svg" alt=""></a>
                     </li>
                     <li>
-                        <a href=""><img src="./images/m.svg" alt=""></a>
+                        <a href=""><img src="./images/m.png" alt=""></a>
                     </li>
                     <li>
                         <a href=""><img src="./images/github.svg" alt=""></a>
                     </li>
                     <li>
                         <a href=""><img src="./images/v.svg" alt=""></a>
-                    </li>
-                    
+                    </li> 
                 </ul>
+        </div>
+    </section>
+    <section>
```

Comment:
Kindly note that as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), the part with the projects should be one section in a `<section>` tag that contains `<article>` s . The section should be styled with `display: grid` and the individual articles styled using `display:flex`
Currently, your project has the articles split into sections. Please fix this.

<img width="1666" alt="Screenshot 2022-10-29 at 10 38 24" src="https://user-images.githubusercontent.com/48546992/198821082-19d2dfae-ed3c-4df8-aded-cd227c65cb19.png">

Reply:
so their should be 2 sections i.e. work section and about-my-self section?